test/unit/test_devices/test_eos_device.py

```diff
@@ -62,10 +63,36 @@ def test_show(self):
 
```

Comment:
These 4 test cases should mock `_parse_response` method and assert that it was called/not called as appropriate

Reply:
I updated them, but only 3 of the 5, the 2 test that test the raise exception I couldn't understand why those would need the mock since it would fail before then.